auth-web/src/components/auth/staff/continuation-in/HomeJurisdictionInformation.vue

```diff
@@ -0,0 +1,387 @@
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { ContinuationReviewFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
+import BusinessService from '@/services/business.services'
+import { CorpTypes } from '@/util/constants'
+import DateUtils from '@/util/date-utils'
+import { JurisdictionLocation } from '@bcrs-shared-components/enums'
+import ModalDialog from '@/components/auth/common/ModalDialog.vue'
+
+@Component({
+  components: {
+    ModalDialog
+  }
+})
+export default class HomeJurisdictionInformation extends Vue {
+  $refs: {
+    errorDialog: InstanceType<typeof ModalDialog>
+  }
+
+  /** Continuation Review object that comes from parent component. */
+  @Prop({ required: true }) readonly continuationReview: ContinuationReviewIF
+
+  // local variables
+  dialogTitle = ''
+  dialogText = ''
+  isDownloading = false
+
+  get continuationIn (): ContinuationReviewFilingIF {
+    return this.continuationReview?.filing?.continuationIn
+  }
+
+  get homeJurisdiction (): string {
+    const region = this.continuationIn?.foreignJurisdiction?.region
+    const country = this.continuationIn?.foreignJurisdiction?.country
+
+    if (country === JurisdictionLocation.CA) {
+      if (region === 'FEDERAL') return 'Federal'
+      const prov = CanJurisdictions.find(can => can.value === region)?.text
+      return (prov || 'Canada')
+    }
+
+    if (country === JurisdictionLocation.US) {
+      const state = UsaJurisdiction.find(usa => usa.value === region)?.text
+      return (state ? `${state}, US` : 'USA')
+    }
+
+    return IntlJurisdictions.find(intl => intl.value === country)?.text || null
+  }
+
+  get identifier (): any {
+    return this.continuationIn?.foreignJurisdiction?.identifier
+  }
+
+  get legalName (): any {
+    return this.continuationIn?.foreignJurisdiction?.legalName
+  }
+
+  get taxId (): any {
+    return this.continuationIn?.foreignJurisdiction?.taxId
+  }
+
+  get incorporationDate (): string {
+    const date = DateUtils.yyyyMmDdToDate(this.continuationIn?.foreignJurisdiction?.incorporationDate)
+    return DateUtils.dateToPacificDate(date, true)
+  }
+
+  get authorizationFiles (): any {
+    return this.continuationIn?.authorization?.files
+  }
+
+  get authorizationDate (): string {
+    const date = DateUtils.yyyyMmDdToDate(this.continuationIn?.authorization?.date)
+    return DateUtils.dateToPacificDate(date, true)
+  }
+
+  /**
+   * Whether a continuation in director affidavit is required.
+   * Is true if the business is a Continued In ULC from Alberta or Nova Scotia.
+   */
+  get isContinuationInAffidavitRequired (): boolean {
+    const entityType = this.continuationIn?.nameRequest?.legalType
+    const country = this.continuationIn?.foreignJurisdiction?.country
+    const region = this.continuationIn?.foreignJurisdiction?.region
+
+    return (
+      entityType === CorpTypes.ULC_CONTINUE_IN &&
+      country === 'CA' &&
+      (region === 'AB' || region === 'NS')
```

Comment:
We don't have constants for countries or regions and I think that's overkill for here.